app/ent/user.go

```diff
@@ -0,0 +1,17 @@
+// Package ent will consist of key entities  of our project
+// and represents core/domain layer of our app -  correct me if I am wrong
+// and would not have any dependencies on other layers.
+// ent package also contains validations and custom errors.
+package ent
```

Comment:
It's better to keep the full name of package, cos it should be meaningful.
Moreover, our full name isn't too long.
So, let's use "entities" instead of "ent"

Reply:
Sure. Thanks. I try to use abbr all the time